cap-ci/config.yaml

```diff
@@ -0,0 +1,30 @@
+jobs:
+  - deploy
+  - smoke-tests
+  - cf-acceptance-tests-brain
+  - cf-acceptance-tests
+  - sync-integration-tests
+  - upgrade
+  - stratos
+backends:
+  - caasp4
+  - aks
+  - gke
+  - eks
+options:
+  - sa
+  - ha
+  - all
+schedulers:
+  - diego
+  - eirini
+
+catapult:
+  uri: https://github.com/SUSE/catapult
+  branch: master
+
+BRAIN:
```

Comment:
We should maintain consistent capitalization and variable casing throughout an individual pipeline. Actually, throughout all our pipelines, but let's start by downcasing this out-of-place capitalization for the key names (even though it needs to be capitalized where it's used for setting the environment within the catapult brains task)

Reply:
DONE.